.github/workflows/_lint.yml

```diff
@@ -70,6 +70,7 @@ jobs:
         working-directory: ${{ inputs.working-directory }}
         run: |
           poetry install --with lint,typing
+          poetry run pip install "pydantic>=1,<2"
```

Comment:
To get lint working. I think we were relying on an optional dep to force pydantic v1 for linting.

Reply:
not necessary